src/cdk/scrolling/virtual-scroll-fixed-size.ts

```diff
@@ -0,0 +1,83 @@
+import {Range} from '@angular/cdk/collections';
+import {Directive, forwardRef, Input, OnChanges} from '@angular/core';
+import {VIRTUAL_SCROLL_STRATEGY, VirtualScrollStrategy} from './virtual-scroll-strategy';
+import {CdkVirtualScrollViewport} from './virtual-scroll-viewport';
+
+
+/** Virtual scrolling strategy for lists with items of known fixed size. */
+export class VirtualScrollFixedSizeStrategy implements VirtualScrollStrategy {
+  private _viewport: CdkVirtualScrollViewport;
+
+  constructor(public itemSize: number, public bufferSize: number) {}
+
+  /** Initialize the strategy and specify the viewport it will be working with. */
+  init(viewport: CdkVirtualScrollViewport) {
```

Comment:
For position and scroll strategies we use `attach` as the method for connecting to the subject of the strategy; we could be consistent with that here.